.github/CODEOWNERS

```diff
@@ -1 +1,6 @@
-* @dequelabs/mobileadmin
\ No newline at end of file
+* @dequelabs/mobile
```

Comment:
BLOCKER: We should remove this line. This line is enforced by the policy on the repo. Having this actually detracts from the value of updating the CodeOwner file.